### PR TITLE
Fix doc typos

### PR DESCRIPTION
**Pull request by chocolateboy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # diff-so-fancy  [![Circle CI build](https://circleci.com/gh/so-fancy/diff-so-fancy.svg?style=shield)](https://circleci.com/gh/so-fancy/diff-so-fancy) [![TravisCI build](https://travis-ci.org/so-fancy/diff-so-fancy.svg?branch=master)](https://travis-ci.org/so-fancy/diff-so-fancy) [![AppVeyor build](https://ci.appveyor.com/api/projects/status/github/so-fancy/diff-so-fancy?branch=master&svg=true)](https://ci.appveyor.com/project/stevemao/diff-so-fancy/branch/master)
 
-`diff-so-fancy` strives to make your diff's **human** readable instead of machine readable. This helps improve code quality and help you spot defects faster.
```
Comment (stevemao) [inline README.md]: diff's is probably correct since it's shortened version of differences? Need native speaker to comment :)

Comment (scottchiefbaker) [reply README.md]: I think he's write, it should be "diffs". It's just a shortened version of differences, and there is no apostrophe in that.

**Comment by scottchiefbaker** [comment]
Looks good to me. Thank you!

File: README.md (hunk 1 above)
Comment (chocolateboy) [reply README.md]: It's not, e.g.:

**Correct**:

- photos
- TVs

**Incorrect**:

- photo's
- TV's

\- but it's a [common mistake](https://en.wikipedia.org/wiki/Apostrophe#Greengrocers%27_apostrophes) (even for native speakers :-)

**Comment by stevemao** [comment]
Thanks!

**Pull request merged**